plugins/quickinspector/quickoverlay.cpp

```diff
@@ -50,6 +50,50 @@ extern Q_GUI_EXPORT QImage qt_gl_read_framebuffer(const QSize &size,
 QT_END_NAMESPACE
 
 namespace GammaRay {
+
+class QQuickItemPropertyCache {
+public:
+    static const QQuickItemPropertyCache &getPropertyCache(QQuickItem *item)
+    {
+        // className() is generated by moc and static -> we can use the pointer as key
```

Comment:
Ok, performance numbers showed that this approach is probably fast enough, and has the benefit of supporting random other types with similar property semantics too. I think we can use QMetaObject* as hash key directly then, rather than the className() pointer, to make this a bit easier to understand?